src/PerfView/PerfViewData.cs

```diff
@@ -3388,6 +3388,436 @@ private void CreateCSVFile(List<AspNetRequest> requests, string csvFileName)
+                            if(request.IsComplete)
+                            {
+                                incompleteRequests.Remove(request.IndexingKey);
+                                completeRequests.Add(request);
+                            }
+                        }
+                        else
+                        {
+                            // request does not exist yet, create and track it
+                            request = new ANCHostingRequest(traceEvent);
+                            request.Method = (string)traceEvent.PayloadByName("method");
+                            request.Path = (string)traceEvent.PayloadByName("path");
+                            if (String.IsNullOrEmpty(request.Path))
+                                request.Path = "/";
+                            request.StartTimeRelativeMSec = traceEvent.TimeStampRelativeMSec;
+                            request.HasStart = true;
+                            incompleteRequests.Add(request.IndexingKey, request);
+                        }
+
+                        break;
+
+                    case _eventRequestStop:
+                        if (incompleteRequests.TryGetValue(ANCHostingRequest.GetIndexingKeyFromEvent(traceEvent), out request))
+                        {
+                            // request exists, update its properties
+                            request.EndTimeRelativeMSec = traceEvent.TimeStampRelativeMSec;
+                            request.HasStop = true;
+
+                            // if the request is complete then move it to the completeRequests List
+                            if (request.IsComplete)
+                            {
+                                incompleteRequests.Remove(request.IndexingKey);
+                                completeRequests.Add(request);
+                            }
+                        }
+                        else
+                        {
+                            // request does not exist yet, create and track it
+                            request = new ANCHostingRequest(traceEvent);
+                            request.EndTimeRelativeMSec = traceEvent.TimeStampRelativeMSec;
+                            request.HasStop = true;
+                            incompleteRequests.Add(request.IndexingKey, request);
+                        }
+
+                        break;
+
+                    case _eventUnhandledException:
+                        if (incompleteRequests.TryGetValue(ANCHostingRequest.GetIndexingKeyFromEvent(traceEvent), out request))
+                        {
+                            // request exists, update it
+                            request.HasUnhandledException = true;
+                        }
+                        else
+                        {
+                            // request does not yet exist, create and track it
+                            request = new ANCHostingRequest(traceEvent);
```

Comment:
If you hit this path, then you don't have any information to associate with the request - is it worth showing this, or should we just leave this request out?

Reply:
It's possible there won't be any other events, but there _should_ be a correlated **Request/Stop** event that comes after. 

However, as of right now, the view does not use this information or surface it in any way to the user. Initially, this was more future-prep for plans when CLR events would be checked and any correlated exceptions can be surfaced with it. I can, however, see a quick enhancement to surface this by way of a boolean output to the request tables, something like an unhandled exception column with a yes/no output. Thoughts?

Reply:
Sure.  Seems reasonable to me.  I think the key is that we want to make it clear to users that the request time is trustworthy (or isn't).  My suspicion is that the number of times we hit this case is low, or the trace is missing lots of events and the data is generally untrustworthy and should be recollected.

Reply:
I have added some output (a table column) to the request tables to indicate via "yes/no" value whether or not a request has an associated unhandled exception and pushed it. Looks like this:
![image](https://github.com/microsoft/perfview/assets/38960254/b2fbf3f5-b978-46fd-9fb1-59474d55bbd8)

Reply:
Looks great!